docs/templates/plugin.rst.j2

```diff
@@ -418,35 +418,6 @@ Status
 - This @{ plugin_type }@ will be removed in version @{ deprecated['removed_in'] | default('') | string | rst_ify }@. *[deprecated]*
 - For more information see `DEPRECATED`_.
 
-{% else %}
-
-{%   set support = { 'core': 'the Ansible Core Team', 'network': 'the Ansible Network Team', 'certified': 'an Ansible Partner', 'community': 'the Ansible Community', 'curated': 'a Third Party'} %}
-{%   set module_states = { 'preview': 'not guaranteed to have a backwards compatible interface', 'stableinterface': 'guaranteed to have backward compatible interface changes going forward'} %}
-
-{%   if metadata %}
-{%     if metadata.status %}
-
-{%       for cur_state in metadata.status %}
-- This @{ plugin_type }@ is @{ module_states[cur_state] }@. *[@{ cur_state }@]*
-{%       endfor %}
-
-{%     endif %}
-
-{%     if metadata.supported_by %}
-{%       set supported_by = support[metadata.supported_by] %}
-- This @{ plugin_type }@ is :ref:`maintained by @{ supported_by }@ <modules_support>`. *[@{ metadata.supported_by }@]*
-
-{%       if metadata.supported_by in ('core', 'network') %}
-Red Hat Support
```

Comment:
Considering all the plugins still in ansible/ansible are supported by us should we just keep this here? Or maybe have it under a conditional var like `supported` that includes this text for "Supported" content that the doc generator can add where necessary.

Reply:
no, docs wont reflect it at all, its been decided from other teams to just use AH is supported, rest is not.

Reply:
Yep but still it would be nice to have the builtin docs for Ansible plugins to show the supported status.